notebooks/official/workbench/exploratory_data_analysis/explore_data_in_bigquery_with_workbench.ipynb

```diff
@@ -847,7 +779,7 @@
       },
       "source": [
         "## Explore event counts by various categories\n",
-        "In this section, you use plotting libraries - specifically [matplotlib](https://matplotlib.org/) and [Plotly Express](https://plotly.com/python/plotly-express/) - to generate graphical outputs that help understand the distribution of events across some of the fields in the sample GA4 data."
+        "In this section, you use plotting libraries - specifically [matplotlib](https://matplotlib.org/) and [Plotly Express](https://plotly.com/python/plotly-express/) - to generate graphical outputs that helps understand the distribution of events across some of the fields in the sample GA4 data."
```

Comment:
add "to"
- "...to generate graphical outputs that helps to understand the distribution..."